cmake/modules/protobuf.cmake

```diff
@@ -47,8 +47,8 @@ else()
 		ExternalProject_Add(protobuf
 			PREFIX "${PROJECT_BINARY_DIR}/protobuf-prefix"
 			DEPENDS zlib
-			URL "https://github.com/protocolbuffers/protobuf/releases/download/v3.17.3/protobuf-cpp-3.17.3.tar.gz"
-			URL_HASH "SHA256=51cec99f108b83422b7af1170afd7aeb2dd77d2bcbb7b6bad1f92509e9ccf8cb"
+			URL "https://github.com/protocolbuffers/protobuf/releases/download/v3.20.3/protobuf-cpp-3.20.3.tar.gz"
```

Comment:
Perhaps i already asked you; is this mandatory to build on loongarch64?

Reply:
Yep, the main reason for the replacement is
protobuf 3.17.3 version cannot be compiled with loongarch, because config.guess and config.sub lack loongarch definition, so it needs to be updated to a higher version

Reply:
Thanks!